packages/hooks/src/useCountDown/demo/demo1.tsx

```diff
@@ -11,7 +11,7 @@ import { useCountDown } from 'ahooks';
 
 export default () => {
   const [countdown, formattedRes] = useCountDown({
-    targetDate: '2022-12-31 24:00:00',
+    targetDate: '2023-12-31 24:00:00',
```

Comment:
What about generating dynamically? u should also change the content inside `<p>` tag.

```suggestion
const targetDate = `${new Date().getFullYear() + 1}-12-31 24:00:00`;
```